pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/AssemblyRenderServlet.java

```diff
@@ -78,41 +77,33 @@ protected void doGet(SlingHttpServletRequest request,
         Module module = request.getResource().adaptTo(Module.class);
         Locale localeObj = LocaleUtils.toLocale(locale);
 
-        Optional<HashableFileResource> moduleVariantSource;
+        Optional<HashableFileResource> moduleVariantSource = null;
 
-        if(draft) {
-            moduleVariantSource = module.moduleLocale(localeObj)
+        moduleVariantSource = module.moduleLocale(localeObj)
                 .traverse()
                 .toChild(ModuleLocale::source)
-                .toChild(SourceContent::draft)
+                .toChild(draft ? SourceContent::draft : SourceContent::released)
                 .getAsOptional();
-        } else {
-            moduleVariantSource = module.moduleLocale(localeObj)
-                    .traverse()
-                    .toChild(ModuleLocale::source)
-                    .toChild(SourceContent::released)
-                    .getAsOptional();
-        }
 
 
         if(!moduleVariantSource.isPresent()) {
             response.sendError(HttpServletResponse.SC_NOT_FOUND, (draft ? "Draft " : "Released ")
                     + "source content not found for " + variantName +  " module variant at "
                     + request.getResource().getPath());
         }
-        else {
-            // collect a list of parameter that traverseFrom with 'ctx_' as those will be used as asciidoctorj
-            // parameters
-            Map<String, Object> context = asciidoctorService.buildContextFromRequest(request);
-
-            // only allow forced rerendering if this is a draft version. Released and historical revs are written in stone.
-            String html = asciidoctorService.getModuleHtml(
-                    module, localeObj, variantName, draft, context, reRender && draft);
-
-            response.setContentType("text/html");
-            Writer w = response.getWriter();
-            w.write(html);
-        }
+
+        // collect a list of parameter that traverseFrom with 'ctx_' as those will be used as asciidoctorj
+        // parameters
+        Map<String, Object> context = asciidoctorService.buildContextFromRequest(request);
+
+        // only allow forced rerendering if this is a draft version. Released and historical revs are written in stone.
+        String html = asciidoctorService.getModuleHtml(
+                module, localeObj, variantName, draft, context, reRender && draft);
+
+        response.setContentType("text/html");
+        Writer w = response.getWriter();
```

Comment:
```suggestion
        response.getWriter().write(html);
```
would eliminate the need for temporary variable.